src/Domain/Model/Address.php

```diff
@@ -140,4 +142,8 @@ public function getCity(): string {
 	public function getCountry(): string {
 		return $this->country;
 	}
+
+	public function isOptedIntoDonationReceipt(): bool {
```

Comment:
From an English language perspective I'm wondering if `hasOptedIntoDonationReceipt` wouldn't be more correct.

Reply:
`is opted into` actually gives the right answer - yes, no. 
What does `has opted into` mean? You always either have true or false, it's never null. :thinking:

Reply:
for me, the user *has* (actively) opted in or out (boolean).
"is" sounds passive, if he "is" opted in or out, *who* opted him in?

Reply:
what's the difference with the address type? The functions are named `isPersonalAddress` and `isCompanyAddress`.

Reply:
An address *is* either a Personal or Company address. But *is* it opted in for a receipt? In my description of a domain, a *user* has opted into receiving a receipt, not the address. 

OTOH, the concept of a "user" (or donor) has not appeared yet in the domain as a description or reference, so an internal variable name of `hasUserOptedIntoReceipt` might be appropriate. 

But we could also leav eout the whole "user" concept and just call it `isOptedIntoDonationReceipt`, because the address change was (i.e. is) opted in for the receipt by someone/something (the user).